lib/puppet/indirector/file_bucket_file/file.rb

```diff
@@ -19,6 +19,8 @@ def find(request)
           other_contents_file = path_for(request.options[:bucket_path], request.options[:diff_with], 'contents')
           raise "could not find diff_with #{request.options[:diff_with]}" unless Puppet::FileSystem.exist?(other_contents_file)
           return `diff #{Puppet::FileSystem.path_string(contents_file).inspect} #{Puppet::FileSystem.path_string(other_contents_file).inspect}`
```

Comment:
Yep it should be `Puppet[:diff]` or `Puppet::Util::Diff.diff` like in `dipper.rb`, but it was already here (Maybe for some obscure reason ...)